entities/manager.rb

```diff
@@ -0,0 +1,156 @@
+class Manager
+  include Taxes
+  include Console
+  include DataUtils
+
+  OPERATIONS = {
+    show_cards: 'SC',
+    create_card: 'CC',
+    destroy_card: 'DC',
+    put_money: 'PM',
+    withdraw_money: 'WM',
+    destroy_account: 'DA'
+  }.freeze
+
+  attr_accessor :current_account
+
+  def initialize
+    @file_path = PATH
+    @current_account = nil
+  end
+
+  def console
+    case start
+    when 'create' then create
+    when 'load' then load
+    else exit
+    end
+  end
+
+  def create
+    name = name_input
+    age = age_input
+    login = login_input(load_accounts.map(&:login))
+    password = password_input
+    @current_account = Account.new(name: name, age: age, login: login, password: password)
+    save_account(@current_account, @file_path)
+    main_menu
+  end
+
+  def show_cards
+    cards_show(@current_account.card)
+  end
+
+  def operate_money(operation)
+    return puts I18n.t(:no_cards) if @current_account.card.none?
+
+    card = card_for_operation(operation)
+    return unless card
+
+    return calculate_withdraw_money(*items_for_operation(card, operation)) if operation == 'withdraw'
+
+    calculate_put_money(*items_for_operation(card, operation))
+  end
+
+  def card_for_operation(operation)
+    puts I18n.t(operation.to_sym)
+    card = choose_card(@current_account.card)
+    card || exit
+  end
+
+  def items_for_operation(card, operation)
+    current_card = @current_account.card[card.pred]
+    money = money_amount(operation)
+
+    [current_card, card, money, withdraw_put_tax(operation, current_card.type, money)]
+  end
+
+  def destroy_account
+    return unless choice_is_yes?(I18n.t(:destroy_account))
+
+    new_accounts = load_accounts.delete_if { |account| account.login == @current_account.login }
+    save_accounts(new_accounts, @file_path)
+  end
+
+  def load
+    accounts = load_accounts
+    return create_the_first_account unless accounts.any?
+
+    login, password = sign_in
+    @current_account = accounts.detect { |account| account.login == login && account.password == password }
+    puts I18n.t(:no_account) unless @current_account
+    main_menu
+  end
+
+  def create_the_first_account
+    return create if choice_is_yes?(I18n.t(:no_accounts))
+
+    console
+  end
+
+  def main_menu
+    choice = main_choices(@current_account.name)
+    return exit if choice == 'exit'
```

Comment:
If I understand correctly, the application will be closed if the user puts exit (according to logic in input method). Why do you need this line?